build.gradle.kts

```diff
@@ -69,5 +71,39 @@ subprojects {
                 junitXml.isOutputPerTestCase = true
             }
         }
+        "uploadArchives"(Upload::class) {
```

Comment:
Are we intentionally uploading every subproject? Any way for them to opt out? Using subprojects is a big gradle antipattern because it makes it very hard to understand how projects are configuring and easy to mess up - that goes doubly for publishing - we don't want to add a new project and have it automatically start publishing to maven central.

Reply:
we can try moving this out to individual sub-projects. I added here to avoid duplicates.